Gemfile.lock

```diff
@@ -157,8 +164,26 @@ GEM
       oauth (>= 0.4.5, < 0.6)
     jbuilder (2.10.1)
       activesupport (>= 5.0.0)
+    jquery-rails (4.4.0)
+      rails-dom-testing (>= 1, < 3)
+      railties (>= 4.2.0)
+      thor (>= 0.14, < 2.0)
+    jquery-ui-rails (6.0.1)
+      railties (>= 3.2.16)
```

Comment:
We use webpack to include these dependencies. However, I was unable to find a way to use these existing dependencies. This seems to be an open issue in RailsAdmin.